etc/katsu.config.example.json

```diff
@@ -97,20 +110,67 @@
                 ]
             }
         },
+        "smoking_status": {
+            "mapping": "individual/extra_properties/smoking_status",
+            "title": "Smoking status",
+            "description": "Smoking status",
+            "datatype": "string",
+            "config": {
+                "enum": [
+                    "Non-smoker",
+                    "Smoker",
+                    "Former smoker",
+                    "Passive smoker",
+                    "Not specified"
+                ]
+            }
+        },
+        "covid_severity": {
+            "mapping": "individual/extra_properties/covid_severity",
+            "title": "Covid severity",
+            "description": "Covid severity",
+            "datatype": "string",
+            "config": {
+                "enum": [
+                    "Uninfected",
+                    "Mild",
+                    "Moderate",
+                    "Severe",
+                    "Dead"
+                ]
+            }
+        },
         "lab_test_result_value": {
             "mapping": "individual/extra_properties/lab_test_result_value",
             "title": "Lab Test Result",
             "description": "This acts as a placeholder for numeric values",
             "datatype": "number",
             "config": {
-                "bins": [200, 300, 500, 1000, 1500, 2000],
+                "bins": [
+                    200,
+                    300,
+                    500,
+                    1000,
+                    1500,
+                    2000
+                ],
                 "minimum": 0,
                 "units": "mg/L"
             }
+        },
+        "experiment_type": {
+            "mapping": "experiment/experiment_type",
+            "mapping_for_search_filter": "individual/biosamples/experiment/experiment_type",
+            "title": "Experiment Types",
+            "description": "Types of experiments performed on a sample",
+            "datatype": "string",
+            "config": {
+                "enum": null
+            }
         }
     },
     "rules": {
         "count_threshold": 5,
         "max_query_parameters": 2
     }
-}
+}
```

Comment:
missing final newline - maybe set your editor to add this automatically

Reply:
my linter actually removed the final newline...

Reply:
weird, since [a trailing newline is required for a file to be a valid "text file" in POSIX](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)